docs/scenarios_authoring/Intro.md

```diff
@@ -108,3 +99,34 @@ The `#meta` variable carries meta information about the currently executed scena
 * properties  
 
 Check [Basic Nodes](BasicNodes#Variable-component) page for examples how to use variables. 
+
+&nbsp;
+## Notion of time | Streaming | Flink engine
```

Comment:
I think `Notion of time` it's fairly complex matter to be in `Introduction`. Maybe it deserves seperate page?